CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+0.5.3 (2019-05-05)
```

Comment:
This change would be after 0.5.3 which was released last year.
(Sorry 0.5.3 didn't make it into the changelog)